lms/views/api/blackboard/sync.py

```diff
@@ -9,10 +9,6 @@ def __init__(self, request):
         self.request = request
         self.grouping_service = self.request.find_service(name="grouping")
```

Comment:
Does this need to be an object any more? This looks a lot like it could be a functional style view and just instantiate this service inline?